tests/testdata/certs/generate.sh

```diff
@@ -35,6 +35,20 @@ subjectAltName = @alt_names
 URI = spiffe://cluster.local/ns/default/sa/default
 EOF
 
+cat > "${WD}/dns-client.conf" <<EOF
+[req]
+req_extensions = v3_req
+distinguished_name = req_distinguished_name
+[req_distinguished_name]
+[ v3_req ]
+basicConstraints = CA:FALSE
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+extendedKeyUsage = clientAuth, serverAuth
+subjectAltName = @alt_names
+[alt_names]
+DNS = server.default.svc
```

Comment:
Shouldn't this dns set for server certificate as well?

Reply:
We are using the same cert for client and server in the test to avoid having to generate so many certs